src/discretize.jl

```diff
@@ -0,0 +1,190 @@
+using LinearAlgebra: inv, rank
+using SparseArrays: spzeros
+
+"""
+    typename(system::AbstractSystem)
+
+    Returns base-type of `system` without information parameter information.
```

Comment:
```suggestion
Returns the base type of `system` without parameter information.
```
Again the indentation is wrong here.

Reply:
🙈